src/App.tsx

```diff
@@ -32,6 +32,13 @@ const App: React.FC = () => {
     setPrecinct('')
     setBallot('')
   }
+  const programCard = () => {
+    fetch('/card/write', {
+      method: 'post',
+      body: JSON.stringify({ code: 'VX.' + ballot + '.' + precinct }),
```

Comment:
`ballot` and `precinct` are in transposed compared to activation codes used in the bmd code: `VX.precinct-23.12D`

Also use template tags > concatenated strings 😄  

```
JSON.stringify({ code: `VX.${ballot}.${precinct}` })
```

Reply:
👍